kso_utils/project.py

```diff
@@ -334,17 +327,13 @@ def connect_zoo_project(self, generate_export: bool = False, zoo_cred=False):
 
     def check_movies_uploaded_zoo(
         self,
-        test: bool = False,
```

Comment:
I've mentioned it before, and you seem to have gotten this yourself. But if you ever see code added to the "real" code, that is specific to the tests, the tests are very likely to be written wrong. Really good you cleaned this up